db/migrate/20151008114125_add_measure_ref_to_chargeback_rate_detail.rb

```diff
@@ -0,0 +1,5 @@
+class AddMeasureRefToChargebackRateDetail < ActiveRecord::Migration
+  def change
+    add_reference :chargeback_rate_details, :chargeback_rate_detail_measure, :foreign_key => true
```

Comment:
Be default the add_reference method create a __:integer__ column. http://apidock.com/rails/v4.0.2/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference 
I am thinking change the add_reference by add_column